src/epjson/epJSONTranslator.cpp

```diff
@@ -77,16 +80,24 @@ const std::string& toJSONFieldName(std::map<std::string, std::string>& fieldName
     return fieldNames.emplace(input, value).first->second;
   };
 
-  if (fieldName == "poissons_ratio") {
-    return cache_value(fieldNameInput, "poisson_s_ratio");
-  }
+  using namespace std::literals::string_view_literals;
+
+  static constexpr std::array<std::pair<std::string_view, std::string_view>, 3> manualFixPairs{{
+    {"poissons_ratio"sv, "poisson_s_ratio"sv},
+    {"youngs_modulus"sv, "young_s_modulus"sv},
+    {"g_function_lnt_ts_value"sv, "g_function_ln_t_ts_value"sv},
+  }};
 
-  if (fieldName == "youngs_modulus") {
-    return cache_value(fieldNameInput, "young_s_modulus");
+  if (auto it = std::find_if(manualFixPairs.cbegin(), manualFixPairs.cend(), [&fieldName](const auto& p) { return p.first == fieldName; });
+      it != manualFixPairs.cend()) {
+    return cache_value(fieldNameInput, it->second);
   }
+
```

Comment:
Add a constexpr map-like so that adding manual field name fixes is easier and clearer (in my opinion)